src/main/java/org/folio/fqm/migration/strategies/V0POCMigration.java

```diff
@@ -411,27 +410,27 @@ public Map<UUID, Function<String, EntityTypeWarning>> getEntityTypeWarnings() {
     return Map.ofEntries(
       Map.entry(
         UUID.fromString("146dfba5-cdc9-45f5-a8a1-3fdc454c9ae2"),
-        fql -> new RemovedEntityWarning("drv_loan_status", "simple_loans", fql)
+        RemovedEntityWarning.withAlternative("drv_loan_status", "simple_loans")
```

Comment:
I refactored all the warnings to use factory methods instead, as having to redeclare a function to pass things around each time was kinda icky